utils/download-bin.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Suggestion: This `utils/download-bin.sh` is a bit of a confusing name. Can we name it something like `download-toolchain.sh`

Reply:
fixed